dist/common_functions.sh

```diff
@@ -140,6 +140,7 @@ check_xargs_P() {
 # Check if the script is run in a recursive mode (with files args).
 # File arguments are those that don't start with a "-".
 is_recursive_mode() {
+  export -n S_RECURSE      # Unexport S_RECURSE to stop propagation to child processes.
```

Comment:
@ershov the PR is merged so no action is required, but I'm curious about how these lines interact:
```
export -n S_RECURSE      # Unexport S_RECURSE to stop propagation to child processes.
[[ -n "${S_RECURSE:-}" ]] && return 0         # S_RECURSE is set.
```
Is the second line now dead code?

Reply:
No, `export -n` "unexports" a variable removing it from the process' environment but the variable still stays in Bash.